README.md

```diff
@@ -158,10 +158,145 @@ This will determine any missing `ansible-core` tags and create them in `ansible-
 nox -s tag
 
 # If you use a different upstream remote, specify the name.
-nox -s tag -- --remote <name> tag 
+nox -s tag -- --remote <name> tag
 
 # If your core repo is not in the same filesystem location, specify the path.
 nox -s tag -- --core <path> tag
 ```
 
 See `nox -s tag -- --help` for extended options.
+
+## Branching for new stable versions
+
+The branching strategy for this repository mirrors the [`ansible/ansible`](https://github.com/ansible/ansible) repository.
+When a new `stable-*` branch is created in the core repository, a corresponding branch in the `ansible-documentation` repository needs to be created.
+There are various other changes that should occur around the same time that the new stable branch is cut.
+
+### Cutting stable branches
+
+Someone with maintainer access must create the new branch.
+
+```bash
+# Make sure your checkout is up to date.
+git fetch upstream
+
+# Create a new stable branch against the devel branch.
+git checkout -b stable-2.18 upstream/devel
+
+# Push the new stable branch to the repository.
+git push upstream stable-2.18:stable-2.18
```

Comment:
Yeah, maybe `<local-branch>:<remote-branch>` is overkill since the branches will have the same name. I did think about the shorter version but wanted to be more explicit.